backend/app/models/log_records.py

```diff
@@ -0,0 +1,41 @@
+from . import db
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+class LogRecords(db.Model):
+    __tablename__ = "log_records"
+    log_id = db.Column(db.Integer, primary_key=True, nullable=False)
+    employee_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+    employee_first_name = db.Column(db.String, nullable=False)
+    employee_last_name = db.Column(db.String, nullable=False)
```

Comment:
since we have the employee table, the first name and last name shouldn't be stored in this table. Rather, the employee_id should be a foreign key that links to the employee table.